src/core_plugins/input_control_vis/public/components/vis/_vis.scss

```diff
@@ -0,0 +1,24 @@
+.icvContainer {
+  width: 100%;
+  margin: 0 $euiSizeXS;
+
+  input[type=number]::-webkit-inner-spin-button,
```

Comment:
What does this do? Guessing it came over from the legacy stuff. Maybe add a comment?

Reply:
It removes this thing 

<img width="82" alt="screen shot 2018-10-04 at 16 09 16 pm" src="https://user-images.githubusercontent.com/549577/46500115-fd903e00-c7ef-11e8-977c-3a1c94b34163.png">


So the input has more room to show numbers, I guess. Yes it was legacy and not worth changing for now.

Reply:
Like I added a comment here: https://github.com/elastic/kibana/pull/23513/commits/e8e7780d0e6399c78d243e5247df65bc12d6cdbc#diff-cae8ca5516d9f6ef6a2f70f54efe7f44

But it's not showing that it was updated..... ugh stupid GH